### PR TITLE
Css fixes

### PR DESCRIPTION
**Pull request by srivig**

### CODE REVIEW

File: db/migrate_cards/data/cards/homepage_company_metric_item [hunk 1]
```diff
@@ -1,25 +1,22 @@
-
+  
   <!--prototype: Company+MetricDesigner+MetricName+yinyang drag item -->
   <div class="metric-item value-item">
     <div>
+      <a class="inherit-anchor" href= "/{{_1|linkname}}?filter[metric]={{_llr+_lr|linkname}}">
```
Comment (ethn) [inline db/migrate_cards/data/cards/homepage_company_metric_item:5]: I think we want to use a cgi_escape view (not linkname) inside the query section of urls.

File: mod/metrics/set/type/wikirate_company/new_answer_form.rb [hunk 2]
```diff
@@ -1,14 +1,65 @@
+include_set Abstract::Table
+
 format :html do
   view :new_metric_value, cache: :never do
     frame do
-      haml_view :new_metric_value_form
+      output [_render_metric_side, _render_source_side]
```
Comment (ethn) [inline mod/metrics/set/type/wikirate_company/new_answer_form.rb]: haml_views are new. Are you confident this doesn't conflict with PK's work?

**Review by ethn — COMMENTED** [review]
I'd like to be confident that the haml_view stuff doesn't mask a real conflict.

File: mod/metrics/set/type/wikirate_company/new_answer_form.rb (hunk 2 above)
Comment (xithan) [reply mod/metrics/set/type/wikirate_company/new_answer_form.rb]: most of the green stuff looks like old code to me

File: db/migrate_cards/data/cards/homepage_company_metric_item (hunk 1 above)
Comment (ethn) [reply db/migrate_cards/data/cards/homepage_company_metric_item:5]: as in `{{_llr+_lr|cgi_escape_name}}`

we might want to alias that with something shorter, like `cgi_name`

File: db/migrate_cards/data/cards/homepage_metric_company_item [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<div class="yinyang-row">
+  <a href="/{{_l|linkname}}?filter[wikirate_company]={{_r|name}}">
```
Comment (ethn) [inline db/migrate_cards/data/cards/homepage_metric_company_item:2]: again, cgi_escape_name here.

File: db/migrate_cards/data/cards/homepage_metric_company_item [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<div class="yinyang-row">
```
Comment (ethn) [inline db/migrate_cards/data/cards/homepage_metric_company_item:1]: just a comment: we're trying to move away from yin-yang row.  We don't have to move quickly, but whenever you work in this area, try to make sure you're using it less and less.

File: db/migrate_cards/data/cards/homepage_metric_company_item [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  <a href="/{{_l|linkname}}?filter[wikirate_company]={{_r|name}}">
+  <div class="company-item value-item">
+    <div class="header no-hover">
+              <div class="logo">
+                      {{_r+image|core;size:small}} 
+              </div>
+          <div class="name">
+              {{_r|name}}
+          </div>
+    </div>
```
Comment (ethn) [inline db/migrate_cards/data/cards/homepage_metric_company_item:11]: bad spacing here.

**Review by ethn — CHANGES_REQUESTED** [review]
please address the cgi escape issue. then I think we're ready to merge.

**Pull request merged**